apps/admin/src/components/sensors/table/sensor-action-cell.tsx

```diff
@@ -66,7 +66,7 @@ export default function SensorActionCell({ sensor }: Props) {
                 <DropdownMenuSeparator />
                 <DropdownMenuItem className="flex cursor-pointer flex-row gap-2" onClick={removeUser}>
                     <MinusIcon className="h-4 w-4" />
-                    Nutzer entfernen
+                    Nutzer reset
```

Comment:
Ich würde sagen, am besten einheitlich bei Deutsch bleiben
```suggestion
                    Nutzer zurücksetzen
```